src/main/java/org/micks/DiscGolfApplication/events/EventRegistrationService.java

```diff
@@ -27,4 +30,29 @@ public void registerUserForEvent(String userId, String eventId) {
             throw new RuntimeException("Error registering user for event", e);
         }
     }
+
+    public List<String> getMyEvents(String userId) {
+        log.info("Fetching event IDs for user: {}", userId);
+        List<String> eventIds = new ArrayList<>();
+
+        try (Connection connection = dbConnection.connect();
+             PreparedStatement statement = connection.prepareStatement(
+                     "SELECT event_id FROM user_event WHERE user_id = ?")) {
+            statement.setString(1, userId);
+
+            try (ResultSet resultSet = statement.executeQuery()) {
+                while (resultSet.next()) {
+                    String eventId = resultSet.getString("event_id");
+                    log.debug("Fetched event ID: {}", eventId);
+                    eventIds.add(eventId);
+                }
+            }
+        } catch (SQLException e) {
+            log.error("Error fetching event IDs for user: {}", userId, e);
```

Comment:
This log is not needed as exception thrown below (RuntimeException) will be logged anyway.

Reply:
Fixed